tracker/tracker_debugger_api.yaml

```diff
@@ -0,0 +1,876 @@
+openapi: 3.0.2
+info:
+  title: Processor Storage API
+  description: Processor Storage API exposes sessions. Handles public endpoint for tracker debugger.
+paths:
+  /api/tracker/v1/debugger:
+    description: Returns a stream of newest sessions.
+    get:
+      parameters:
+        - name: Authorization
+          description: JSON Web Token
+          schema:
+            type: string
+            example: 'Bearer valid.and_propper.jwt_value'
+          in: header
+          required: true
+        - name: app_id
+          description: An application UUID for which sessions are requested
+          schema:
+            type: string
+            example: 'd898ae47-61cd-4c9d-9d67-e5a930eb9f38'
+          in: query
+          required: true
+        - name: limit
+          description: 'Limit of fetched sessions'
+          schema:
+            type: integer
+            example: 20
+          in: query
+          required: false
+        - name: offset
+          description: Fetched sessions offset
+          schema:
+            type: integer
+            example: 100
+          in: query
+          required: false
+        - name: ip
+          description: IP address from which event originated
+          schema:
+            type: string
+            example: '73.112.8.189'
+          in: query
+          required: false
+        - name: visitor_id
+          description: Visitor ID of the user
+          schema:
+            type: string
+            example: '288f90221c221f81'
+          in: query
+          required: false
+        - name: user_id
+          description: User ID dimension of the user
+          schema:
+            type: string
+            example: 'my_user_login'
+          in: query
+          required: false
+        - name: lookup_window
+          description: 'Describes how much back in time sessions shall be fetched, takes value in minutes, max 300'
+          schema:
+            type: string
+            default: 30
+            example: 90
+          in: query
+          required: false
+      responses:
+        '200':
+          content:
+            application/x-ndjson:
+              schema:
+                type: array
```

Comment:
NDJSON is not array, this is what differentiates it from array of JSON objects.

Reply:
As I can see we describe this type as an array also in our specification in PSA, I hope this won't cause any misunderstanding.